Source/CombatExtended/CombatExtended/Verbs/Verb_LaunchProjectileCE.cs

```diff
@@ -996,7 +996,7 @@ protected virtual bool KeepBurstOnNoShootLine(bool suppressing, out ShootLine sh
             // 5:       Interruptible -> stop shooting
             // 6:       Not interruptible -> shoot along previous line
             // 7:     else -> stop
-            shootLine = (ShootLine)lastShootLine;
+            shootLine = lastShootLine ?? default;
```

Comment:
```suggestion
            if (lastShootLine == null)
            {
                shootLine = default;
                return false;
            }
```
We don't need futher calculations if we have no lastShootLine